chapter-2/README.zh-cn.md

```diff
@@ -52,6 +52,10 @@ EOF
 
 脚本运行结束后，所有必要的镜像都会被拉取到本地并被加载到 Kind 集群的所有节点之中。如果你使用的是运供应商的 Kubernetes 集群，因为带宽足够，可能就没有必要使用这个脚本了。
 
+**注意：** 如果你是在 MacOS 运行的 Docker Desktop，请保证其虚拟磁盘至少有 15Gb 的可用空间。你可以在 Settings -> Resources 菜单栏修改 Virtual Disk limit 的值。
```

Comment:
@zouyingjie same here.. I would mention a number of GigaBytes here.. I would keep it generic.

Reply:
Updated.